sync/worker.go

```diff
@@ -140,6 +141,8 @@ func NewBlockWorker(s *Syncer, count int, reqFactory BlockRequestFactory, ids ch
 				}
 			}
 			if !retrived {
+				// for now panic if the timeout is not enough for all peers
```

Comment:
what should happen here?

Reply:
@barakshani 
@almogdepaz 
@zalmen 
What should we do (assuming we have no "refresh peers")
Please note I think that it doesn't happen after the fix and after we use a decent timeout.
The panic only failed the UT but if that is the behavior we want I can work on that.

Reply:
here you should do nothing

Reply:
you shouldn't panic here anyway this is just the fetcher the receiving side should decide what to do on error, any way this will all be handled in the validation queue